README.md

```diff
@@ -22,7 +22,7 @@
   <dd>Example tests showcasing mocking, assertions and configuration for mainnet forking. As well it includes everything needed in order to check code coverage.</dd>
 
   <dt>Linter</dt>
-  <dd>Linter for every language present in the repository, including solidity by using <a href="https://github.com/protofire/solhint">solhint</a>. As well, linter runs only against staged git files by using <a href="https://github.com/okonet/lint-staged">lint-staged</a>.</dd>
+  <dd>Simple and fast solidity linting by using forge fmt</a>.</dd>
```

Comment:
by using reads slightly weird to me, maybe "through the use of" or "thanks to forge fmt"